clang/lib/CIR/CodeGen/CIRGenExprConst.cpp

```diff
@@ -1338,7 +1337,7 @@ mlir::Attribute ConstantEmitter::tryEmitPrivateForVarInit(const VarDecl &D) {
   }
   InConstantContext = D.hasConstantInitialization();
 
-  const Expr * E = D.getInit();
+  const Expr *E = D.getInit();
```

Comment:
The last 3 changes are all unrelated to this PR, please remove them. There are others after this one coming up in the patch, you should also remove them. `LowerToLLVM.cpp` has lots of these too.

I'll wait until you remove all the noise before I take another look into this PR, thanks!